communication/README.md

```diff
@@ -188,7 +188,7 @@ place!
 [Discuss Kubernetes]: https://discuss.kubernetes.io
 [Join]: http://slack.k8s.io
 [Slack Guidelines]: /communication/slack-guidelines.md
-[10am US Pacific Time]: https://www.google.com/search?q=1000+am+in+pst
+[10am US Pacific Time]: https://www.google.com/search?q=1000+am+in+pacific+time
```

Comment:
```suggestion
[10am US Pacific Time]: https://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29
```
Both the google links display the wrong timezone (pics below) >_< 
The suggestion here is the preferred tz converter per the [doc style guide](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#dates-and-times).


<img width="642" alt="Screen Shot 2021-03-21 at 9 57 50 AM" src="https://user-images.githubusercontent.com/6500863/111907546-45d96000-8a2c-11eb-909c-a5dbed7ded2f.png">
<img width="610" alt="Screen Shot 2021-03-21 at 9 57 25 AM" src="https://user-images.githubusercontent.com/6500863/111907548-470a8d00-8a2c-11eb-92f6-9f279edf5031.png">

Reply:
Thanks!